build.sbt

```diff
@@ -1,33 +1,33 @@
 val catsParseVersion           = "0.3.10"
```

Comment:
I think we can update `cats-parse` since you updated core.

Reply:
The latest tagged core doesn't include that change, so I'll leave it for now.